components/group/GroupMembers.js

```diff
@@ -396,7 +396,7 @@ const GroupMembers = ({ group, accessToken, reloadGroup }) => {
     readers: [profileId],
     writers: [profileId],
     signatures: [profileId],
-    invitation: group.domain ? `${group.domain}/-/Edit` : group.invitations[0],
+    invitation: group.domain ? `${group.domain}/-/Edit` : group.invitations?.[0],
```

Comment:
We should use `OpenReview.net/-/Username` and `OpenReview.net/-/Email` for tilde groups and email groups respectively instead of the `OpenReview.net/-/Edit` meta invitation.